cmd/minikube/cmd/docker-env.go

```diff
@@ -172,6 +182,17 @@ var dockerEnvCmd = &cobra.Command{
 			}
 		}
 
+		out, err := tryDockerConnectivity("docker", ec)
+		if err != nil { // docker might be up but been loaded with wrong certs/config
+			glog.Warningf("couldn't connect to docker inside minikube. output: %s error: %v", string(out), err)
+			// to fix issues like this #8185
+			glog.Infof("will try to restart dockerd service...")
```

Comment:
Can this be combined with the warning message? Users won't see the info message by default, leaving them with a scary warning and no explanation that we are trying to resolve it.

Reply:
done